### ISSUE TITLE
Updating GENCODE parser

### ISSUE THREAD

**Issue by DSuveges** [DSuveges/GenomePlotter#17]
More information is required in the final gencode file:

- [x] Adding direction of gene `+` or `-`
- [x] Adding CDS

**Comment by DSuveges** [comment]
As a resolution, two files are generated based on gencode data. One is the old and new, with the following columns:

```
chr    start     end       strand  type  gene_id          gene_name
chr17  58083419  58083683  +       UTR   ENSG00000264364  DYNLL2
chr17  58087082  58087090  +       UTR   ENSG00000264364  DYNLL2
chr17  58087091  58087222  +       CDS   ENSG00000264364  DYNLL2
chr17  58089142  58089276  +       CDS   ENSG00000264364  DYNLL2
chr17  58089277  58095542  +       UTR   ENSG00000264364  DYNLL2
```

These are all the annotations we need for the generation of the arrows plot.

**Comment by DSuveges** [comment]
Done. Addressed at #19
